region/norfair/east.json

```diff
@@ -4645,8 +4650,13 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
+                    "Gravity",
                     {"heatFrames": 375},
-                    {"spikeHits": 2}
+                    {"spikeHits": 2},
+                    {"or": [
+                      "canIframeSpikeJump",
+                      {"spikeHits": 2}
```

Comment:
or canDamageBoost tbh

Reply:
A note about turning off speed would be nice too while we're here.  I feel it comes up the most in this room.